src/main/java/io/lighty/yang/validator/formats/yang/printer/ModulePrinter.java

```diff
@@ -301,7 +296,7 @@ private void doPrintSchema(final boolean isPrintingAllowed, final SchemaTree tre
         if (isPrintingAllowed) {
             if (schemaNode instanceof ContainerLike) {
                 printer.openStatement(Statement.CONTAINER, schemaNode.getQName().getLocalName());
-                printer.printConfig(schemaNode.isConfiguration());
+                printer.printConfig(schemaNode.effectiveConfig().orElse(Boolean.TRUE));
```

Comment:
No, we do not change isConfiguration().

Reply:
It's marked for removal. This will need to be done soon. Should I remove it?

Reply:
No. Yes.

Reply:
Done